packages/api/src/services/repository/repository-rest.service.ts

```diff
@@ -1,9 +1,18 @@
 import {Service} from '../../providers/service/service';
+import {Repository, RepositorySizeInfo} from '../../models/repositories';
 
 export class RepositoryRestService implements Service {
-  getRepositories(): Promise<Record<string, unknown[]>> {
+
+  static readonly REPOSITORIES_ENDPOINT = 'rest/repositories';
+
+  getRepositories(): Promise<Record<string, []>> {
```

Comment:
Wasn't there a concrete type for the repository list? Use it!

Reply:
Done